### PR TITLE
make installable

### PR DESCRIPTION
**Pull request by philippeller**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -59,8 +59,9 @@
     packages=find_packages(),
     include_dirs=[np.get_include()],
     package_data={
-        'retro.tables': '*.pyx'
```
Comment (jllanfranchi) [inline setup.py]: My mistake: I was thinking "pyc" when I said you shouldn't include "pyx", but we _do_ want "pyx" files (these are cython files)

**Comment by philippeller** [comment]
Can this be merged?

**Comment by jllanfranchi** [comment]
:+1:

**Comment by jllanfranchi** [comment]
yes

**Pull request merged**